ext/ctype/ctype.stub.php

```diff
@@ -2,24 +2,51 @@
 
 /** @generate-class-entries */
 
+/**
+ * @compile-time-eval
```

Comment:
All ctype functions are locale-dependent, not just ctype_lower/upper.

Reply:
reverted

according https://stackoverflow.com/questions/36592540/unable-to-set-utf-16-as-locale locale has to be ASCII compatible, so are functions like `ctype_digit` and `ctype_xdigit` really locale dependent?